src/interchange/src/json.rs

```diff
@@ -77,7 +78,13 @@ impl fmt::Debug for JsonEncoder {
                 "schema",
                 &format!(
                     "{:?}",
-                    build_row_schema_json(&self.value_columns, "schema", &BTreeMap::new(), false)
+                    build_row_schema_json(
```

Comment:
Uncovered

Reply:
This is the `Debug` implementation only applicable when this object will be logged and I am guessing we don't log this anywhere, which is probably fine.